src/main/java/core/basesyntax/suppliers/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.suppliers;
```

Comment:
I wouldn't say that this is a supplier, this is more of a figures part, since it is part of the model class